modules/tinymce/src/core/test/ts/browser/focus/KeyboardShortcutMenuFocusTest.ts

```diff
@@ -0,0 +1,46 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { McEditor, TinyContentActions, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import { RawEditorOptions } from 'tinymce/core/api/OptionTypes';
+
+describe('browser.tinymce.themes.silver.editor.toolbar.KeyboardShortcutMenuFocusTest', () => {
+
+  const pTestFocus = async (options: RawEditorOptions) => {
```

Comment:
Similar to the previous test, please update this so the context/it blocks are swapped and then use `Arr.each` for the separate configuration.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/7c91f0e22441844e9236c7393bd8e514e39968a0